scripts/lib.sh

```diff
@@ -0,0 +1,11 @@
+#
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+die()
```

Comment:
why not use the `die` function in `.ci/lib.sh`?

Reply:
I've moved the script to `.ci/` now. I still think we might want to split out some of what's in `.ci/` to a `scripts` directory but for the time being let's keep all the scripts together.